camelot/view/controls/delegates/customdelegate.py

```diff
@@ -134,28 +135,32 @@ def __init__(self, parent=None, editable=True, **kwargs):
         self._width = self._font_metrics.averageCharWidth() * 20
 
     @classmethod
-    def get_standard_item(cls, locale, value, field_attributes_values):
+    def get_standard_item(cls, locale, model_context):
         """
         This method is used by the proxy to convert the value of a field
         to the data for the standard item model.  The result of this call can be
         used by the methods of the delegate.
 
         :param locale: the `QLocale` to be used to display locale dependent values
-        :param value: the value of the field on the object
-        :param field_attributes_values: the values of the field attributes on the
-           object
+        :param model_context: a FieldActionModelContext object
         
         :return: a `QStandardItem` object
         """
+        action_states = []
+        for action in model_context.field_attributes.get('actions', []):
```

Comment:
note : er is ook een 'action_routes' field attribute die een unieke string per actie bevat, dmv een
zip(action_routes, actions) zouden we de state in een dict kunnen steken met een action_route als key ...
moest dat nodig zijn om aan de ui kant de states er weer uit te kunnen vissen voor de juiste action